lib/srv/desktop/tdp/conn.go

```diff
@@ -117,3 +118,17 @@ func (c *Conn) LocalAddr() net.Addr {
 func (c *Conn) RemoteAddr() net.Addr {
 	return c.remoteAddr
 }
+
+// IsNonFatalErr returns whether or not an error arising from
+// the tdp package should be interpreted as fatal or non-fatal
+// for an ongoing TDP connection.
+func IsNonFatalErr(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	return errors.Is(err, trace.LimitExceeded(clipDataMaxLenErr)) ||
```

Comment:
Are there limit exceeded errors that are fatal?

If not, how about just `return trace.IsLimitExceeded(err)`.

Reply:
I think it's better to limit this to specific error messages so that we don't have to always remember that `trace.LimitExceeded` will be counted as non-fatal.